src/System.Buffers/src/Resources/Strings.resx

```diff
@@ -117,4 +117,25 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="InvalidDoubleReturn" xml:space="preserve">
+    <value>The specified Buffer has already been returned to the pool.</value>
+  </data>
```

Comment:
This is used in the UT that is commented out for right now; once we determine the correct course of action for the UT, I'll make the proper changes to this string :)